tpgtools/overrides/cloudresourcemanager/beta/folder.yaml

```diff
@@ -0,0 +1,4 @@
+- type: SERIALIZATION_ONLY
+- type: CUSTOM_TERRAFORM_PRODUCT_NAME
+  details:
+    product: ""
```

Comment:
Maybe a comment that says "this produces the correct name `google_folder`"?

Reply:
Done